### PR TITLE
SPCTR-2382 - Improvements: Custom URL PO Suggestions.

### PR DESCRIPTION
**Pull request by bsf-zanev**
### Description
Added PO Suggestions into the Custom URL feature.

### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?
Tested as per the cases in the associated card.

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: blocks-config/image-gallery/class-spectra-image-gallery.php [hunk 1]
```diff
@@ -1054,7 +1058,7 @@ private function render_lightbox( $attributes ) {
 								<img class="swiper-lazy" data-src="<?= esc_url( $attributes['mediaGallery'][ $i ]['url'] ); ?>" alt="<?= esc_attr( $attributes['mediaGallery'][ $i ]['alt'] ) ?>"/>
 								<div class="swiper-lazy-preloader swiper-lazy-preloader-white"></div>
 								<?php if ( $attributes['lightboxDisplayCaptions'] ) : ?>
-									<div class="spectra-image-gallery__control-lightbox--caption">
+									<div class="spectra-image-gallery__control-lightbox--caption" data-spectra-gallery-image-id='<?= esc_attr( $attributes['mediaGallery'][ $i ]['id'] ) ?>'>
```
Comment (adi3890) [inline blocks-config/image-gallery/class-spectra-image-gallery.php:1061]: please check for shorthand tags

Comment (bsf-zanev) [reply blocks-config/image-gallery/class-spectra-image-gallery.php:1061]: Done this, will push in the updated comment.

File: includes/blocks/image-gallery/frontend.css.php [hunk 2]
```diff
@@ -756,9 +756,9 @@
 $selectors[' .spectra-image-gallery__layout--carousel .slick-dots']['margin-bottom'] = '30px !important';
 
 $combined_selectors = array(
-	'desktop' => $selectors,
-	'tablet'  => $t_selectors,
-	'mobile'  => $m_selectors,
+	'desktop' => apply_filters( 'uagb_image_gallery_styling', $selectors, $attr ),
```
Comment (adi3890) [inline includes/blocks/image-gallery/frontend.css.php:759]: please use UAGB_Helper::get_combined_selectors
reference https://github.com/brainstormforce/ultimate-addons-for-gutenberg/blob/e63afcf7f8f19caffda79a2cc2e7ed1738c5e4c1/includes/blocks/advanced-heading/frontend.css.php#L273

Comment (bsf-zanev) [reply includes/blocks/image-gallery/frontend.css.php]: Added the UAGB Helper Method.

**Review by RajkiranBagal — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**